app/classes/map_set.rb

```diff
@@ -119,10 +119,7 @@ def east_west_distance
   def update_extents_with_point(loc)
     lat = loc.lat.to_f.round(4)
     long = loc.long.to_f.round(4)
-    if !@north
-      @north = @south = lat
-      @east = @west = long
-    else
+    if @north
```

Comment:
Here (and in many other places), the `if` and `else` code is swapped in order to use a positive condition with `if`.
All of these have been autocorrected, so there won't be any transcription or other human errors. 
Rubocop says the cop and autocorrection are safe, which makes sense to me.
See Style/NegatedIfElseCondition, https://docs.rubocop.org/rubocop/1.31/cops_style.html#stylenegatedifelsecondition